dojo/templates/dojo/view_finding.html

```diff
@@ -556,6 +556,9 @@ <h3 class="pull-left finding-title">
                 {% if finding.component_version %}
                 <th>Component Version</th>
                 {% endif %}
+                {% if finding.fix_version %}
+                <th>Fixed Version</th>
+                {% endif %}
```

Comment:
I think we should add the fix_available field as well? Maybe combine it into the same cell, i.e. "Fix available": Yes (4.13.4)

Reply:
But where is the fix_available field shown?

Reply:
The fix_available might be interesting to show also when it's False or Unknown (None)